execution/pegged_orders.go

```diff
@@ -155,7 +152,7 @@ func (p *PeggedOrders) RemoveAllParkedForParty(
 
 	for _, o := range p.orders {
 		if o.Party == party && o.Status == types.OrderStatusParked {
-			o.UpdatedAt = p.currentTime
+			o.UpdatedAt = p.timeService.GetTimeNow().UnixNano()
```

Comment:
same, can do it once outside the loop

Reply:
Addressed here https://github.com/vegaprotocol/vega/pull/5475#discussion_r904949850